pkgs/applications/misc/keymapp/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, wrapGAppsHook
+, libusb1
+, webkitgtk
+, gtk3
+, writeShellScript
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keymapp";
+  version = "1.0.5";
+
+  src = fetchurl {
+    url = "https://oryx.nyc3.cdn.digitaloceanspaces.com/keymapp/keymapp-${version}.tar.gz";
+    hash = "sha256-e9Ty3gMb+nkXGK8sND4ljyrIxP+1fLasiV6DoTiWmsU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    wrapGAppsHook
```

Comment:
adding this without further changes does result in a double wrapped application, as its creating a wrapper here already.
We can not use the bashWrapper at all (and can ship ) `__NV_PRIME_RENDER_OFFLOAD` as `--prefix` per `gappsWrapperArgs` or do vice versa and tell  `wrapGAppsHook` not to create the wrapper by setting  `dontWrapGApps = true;` and feeding `gappsWrapperArgs[@]` to the bash wrapper, to i am still unsure to set the variable at all, as there are not really occurences of that in nixpkgs beside for nvidia_offload and gamescope. Maybe someone else with a nvidia card can be a additional source of knowledge here how to handle that (i just have amd cards available).